.github/workflows/deploy.yaml

```diff
@@ -18,6 +18,8 @@ jobs:
       uses: actions/checkout@v3
     - name: Setup python
       uses: actions/setup-python@v4
+      with:
+        python-version: 3.10
```

Comment:
I used in another repo 3.x to not worry about this too much, but probably 3.10 is good (just need to remember to increase this from time to time)